db/seeds/users.rb

```diff
@@ -1,2 +1,188 @@
+User.create(first: 'Cake', last: 'Roll' , email: 'CakeRoll@Roll.com', user_name: 'CakeRoll', street: '88699 Main St.' , city: 'Miami', state: 'FL', zip: 66666, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Muffin', last: 'Roll' , email: 'MuffinRoll@Roll.com', user_name: 'MuffinRoll', street: '42986 Main St.' , city: 'Topeka', state: 'KS', zip: 94625, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Pastry', last: 'Roll' , email: 'PastryRoll@Roll.com', user_name: 'PastryRoll', street: '67082 Main St.' , city: 'Paris', state: 'TX', zip: 44444, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Macaroon', last: 'Souffle' , email: 'MacaroonSouffle@Souffle.com', user_name: 'MacaroonSouffle', street: '28809 Main St.' , city: 'Paris', state: 'TX', zip: 44444, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Donut', last: 'Souffle' , email: 'DonutSouffle@Souffle.com', user_name: 'DonutSouffle', street: '55088 Main St.' , city: 'Springfield', state: 'MO', zip: 57294, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Icing', last: 'Souffle' , email: 'IcingSouffle@Souffle.com', user_name: 'IcingSouffle', street: '40536 Main St.' , city: 'Springfield', state: 'MO', zip: 57294, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Lollipop', last: 'Souffle' , email: 'LollipopSouffle@Souffle.com', user_name: 'LollipopSouffle', street: '81077 Main St.' , city: 'Miami', state: 'FL', zip: 66666, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Wafer', last: 'Souffle' , email: 'WaferSouffle@Souffle.com', user_name: 'WaferSouffle', street: '19795 Main St.' , city: 'Nashua', state: 'NH', zip: 13062, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Pudding', last: 'Sugar' , email: 'PuddingSugar@Sugar.com', user_name: 'PuddingSugar', street: '96121 Main St.' , city: 'Newtown', state: 'CT', zip: 16474, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Donut', last: 'Sugar' , email: 'DonutSugar@Sugar.com', user_name: 'DonutSugar', street: '33 Main St.' , city: 'Nashua', state: 'NH', zip: 13062, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Cotton', last: 'Sugar' , email: 'CottonSugar@Sugar.com', user_name: 'CottonSugar', street: '39887 Main St.' , city: 'Nashua', state: 'NH', zip: 13062, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Candy', last: 'Sugar' , email: 'CandySugar@Sugar.com', user_name: 'CandySugar', street: '98404 Main St.' , city: 'Topeka', state: 'KS', zip: 94625, password: 'password', password_confirmation: 'password', admin: false)
```

Comment:
Line is too long. [238/80]<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.